Upload wpt local-network-access tests

igower opened the pull request:
#### 04a08c1b1d1aaf8b59f258861d950127060b6068
<pre>
Upload WPT Local-Network Access Tests
<a href="https://bugs.webkit.org/show_bug.cgi?id=295049">https://bugs.webkit.org/show_bug.cgi?id=295049</a>
<a href="https://rdar.apple.com/154416306">rdar://154416306</a>

Reviewed by NOBODY (OOPS!).

* LayoutTests/imported/w3c/web-platform-tests/fetch/local-network-access/README.md:
* LayoutTests/imported/w3c/web-platform-tests/fetch/local-network-access/iframe.tentative.https.window.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/fetch/local-network-access/iframe.tentative.https.window.js: Added.
(setup):
(makePermissionTests):
* LayoutTests/imported/w3c/web-platform-tests/fetch/local-network-access/navigate.tentative.https.window.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/fetch/local-network-access/navigate.tentative.https.window.js: Added.
(setup):
* LayoutTests/imported/w3c/web-platform-tests/fetch/local-network-access/navigate.tentative.window.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/fetch/local-network-access/navigate.tentative.window.js: Added.
(setup):
* LayoutTests/imported/w3c/web-platform-tests/fetch/local-network-access/resources/iframer.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/fetch/local-network-access/resources/navigate.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/fetch/local-network-access/resources/openee.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/fetch/local-network-access/resources/support.sub.js:
(Server):
* LayoutTests/imported/w3c/web-platform-tests/fetch/local-network-access/resources/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/fetch/local-network-access/w3c-import.log:
* Tools/CISupport/build-webkit-org/steps.py:
(CompileWebKit.parseOutputLine):
</pre>

brentfulgham reviewed (approved):
This looks right to me, but we should get @nt1m to confirm we haven't forgotten something test related.

igower committed:
Upload wpt local-network-access tests
https://bugs.webkit.org/show_bug.cgi?id=295049
rdar://154416306

Reviewed by Brent Fulgham and Tim Nguyen.

Uploaded tests from wpt
Upstream commit: https://github.com/web-platform-tests/wpt/commit/129b2b90e28996ed1bf234f7efd5da998f18474f

* LayoutTests/TestExpectations:
* LayoutTests/imported/w3c/web-platform-tests/fetch/local-network-access/META.yml: Added.
* LayoutTests/imported/w3c/web-platform-tests/fetch/local-network-access/README.md: Added.
* LayoutTests/imported/w3c/web-platform-tests/fetch/local-network-access/fetch.tentative.https-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/fetch/local-network-access/fetch.tentative.https.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/fetch/local-network-access/resources/fetch-local-http.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/fetch/local-network-access/resources/fetch-local.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/fetch/local-network-access/resources/fetch-public-http-wrong-address-space.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/fetch/local-network-access/resources/support.sub.js: Added.
(Server.get const):
(Server):
(resolveUrl):
(sourceResolveOptions):
* LayoutTests/imported/w3c/web-platform-tests/fetch/local-network-access/resources/target.py: Added.
(_get_response_headers):
(_is_loaded_in_fenced_frame):
(_final_response_body):
(_handle_final_request):
(main):
* LayoutTests/imported/w3c/web-platform-tests/fetch/local-network-access/resources/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/fetch/local-network-access/w3c-import.log: Added.

Canonical link: https://commits.webkit.org/296691@main
